at_onboarding_flutter/lib/screen/at_onboarding_qrcode_screen.dart

```diff
@@ -73,49 +74,50 @@ class _AtOnboardingQRCodeScreenState extends State<AtOnboardingQRCodeScreen> {
     // we need to listen for Flutter SizeChanged notification and update controller
     return QRView(
       key: qrKey,
+      cameraFacing: CameraFacing.back,
       onQRViewCreated: _onQRViewCreated,
+      formatsAllowed: const [BarcodeFormat.qrcode],
       overlay: QrScannerOverlayShape(
-          borderColor: Theme.of(context).primaryColor,
-          borderRadius: 10,
-          borderLength: 30,
-          borderWidth: 10,
-          cutOutSize: scanArea),
-      onPermissionSet: (ctrl, p) => _onPermissionSet(context, ctrl, p),
+        borderColor: Theme.of(context).primaryColor,
+        borderRadius: 10,
+        borderLength: 30,
+        borderWidth: 10,
+        cutOutSize: scanArea,
+      ),
     );
   }
 
   void _onQRViewCreated(QRViewController controller) {
     setState(() {
-      this.controller = controller;
-    });
-    controller.scannedDataStream.listen((scanData) {
-      onScannedData(scanData.code);
+      _controller = controller;
     });
-  }
 
-  void _onPermissionSet(BuildContext context, QRViewController ctrl, bool p) {
-    if (!p) {
```

Comment:
Are we not showing any error message when camera permission is not granted ?

Reply:
done